src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -59,11 +60,12 @@ public class ExprCommand extends SimpleExpression<String> {
 	}
 
 	private boolean fullCommand;
+	private boolean SUPPORTS_UNKNOWN_EVENT = Skript.classExists("org.bukkit.event.command.UnknownCommandEvent");
 	
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```

Comment:
```suggestion
	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
```
and newline after matchPattern